src/plugins/intel_cpu/src/utils/jit_kernel.cpp

```diff
@@ -33,7 +33,7 @@ void freeReg(jit_kernel::reg_indices & freeRegs, const registers<RegType> & regs
     //     throw std::runtime_error("Some register was freed twice");
     freeRegs.emplace_back(idx);
     if (freeRegs.size() > regs.size())
-        IE_THROW() << "Some register was freed twice";
+        throw std::runtime_error("Some register was freed twice");
```

Comment:
I guess it was changed on IE_THROW in the previous PR and now it is reverted. Why? What exactly coverity "says"?